src/reducer.js

```diff
@@ -9,6 +9,11 @@ function reducer(
         fetchedBatchRunPicker: false,
         errorBatchRunPicker: null,
         batchRunPicker: [],
+        fetchingBatchRunWithLocationPicker: false,
+        fetchedBatchRunWithLocationPicker: false,
+        batchRunWithLocationPickerPageInfo: {},
```

Comment:
If you use `formatPageQuery` instead of `formatPageQueryWithCount` then you do not need `batchRunWithLocationPickerPageInfo` as well as there are no pages.